src/components/subreddits/index.js

```diff
@@ -3,14 +3,30 @@ import React from 'react';
 import Subreddit from './item';
 
 export default class SubredditsContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selected: null
+    }
+  }
+
+  onChildClick(selected) {
+    this.setState({
+      selected: selected
+    });
+  }
+
   renderSubreddits(subreddits) {
     return subreddits.map(item => {
       return (
         <Subreddit
           activate={this.props.setActiveSubreddit}
           key={item.data.id}
           name={item.data.display_name}
-          url={item.data.url} />
+          url={item.data.url}
+          selected={this.state.selected==item.data.display_name}
```

Comment:
for javascript, we use 3 equal signs for equality. Also, you should put spaces there like:
`this.state.selected === item.data.display_name`